lib/facter/resolvers/networking.rb

```diff
@@ -15,6 +15,7 @@ def post_resolve(fact_name)
         def read_facts(fact_name)
           interfaces_data
           primary_interface
+          primary6_interface
```

Comment:
This resolver is used by Osx and FreeBsd. Not sure it will work and you don't have the primary6 facts for these operating systems.

Reply:
I think that's OK, facter should just return empty string like for any other nonexistent or unresolvable fact.

Reply:
Well then at least you should add an implementation for the `primary6_interface` method. But we shouldn't add data that is not used by any fact.

Reply:
Removed this so that it's a Linux-only fact for now.